packages/terra-form-radio/src/Radio.jsx

```diff
@@ -52,21 +52,26 @@ const propTypes = {
     */
   name: PropTypes.string,
   /**
-   * Function to trigger when focus is lost from the radio button.
-   */
+    * Function to trigger when focus is lost from the radio button.
+    */
   onBlur: PropTypes.func,
   /**
     * Function to trigger when user clicks on the radio button. Provide a function to create a controlled input.
     */
   onChange: PropTypes.func,
   /**
-   *  Function to trigger when user focuses on the radio button.
-   */
+    *  Function to trigger when user focuses on the radio button.
+    */
   onFocus: PropTypes.func,
   /**
     * The value of the input element.
     */
   value: PropTypes.string,
+  /**
+    * @private
+    * Function to trigger on key press on the radio button. Private functions created to be used by field component.
+    */
+  onKeyDown: PropTypes.func,
```

Comment:
We should add this prop to the CHANGELOG under the `Added` section with a summary of its usage.
e.g.
```
* Added
  * Added (name) prop to (usage). 
```